Dockerfile

```
@@ -7,12 +7,13 @@ RUN apt update && apt install -y curl unzip libsnappy-dev build-essential jq
 COPY . /runner
 WORKDIR /runner
 
-ENV WEB_SERVER gunicorn
+ENV WEB_SERVER_TYPE gunicorn
+ENV WEB_SERVER_WORKER_CLASS gevent
 ENV WEB_SERVER_WORKERS 3
-ENV GUNICORN_KEEP_ALIVE 2
-ENV GUNICORN_CMD_ARGS -c gunicorn_config.py
 ENV WEB_SERVER_THREADS 10
 ENV WEB_SERVER_ASYNC_CORES 10
+ENV HTTP_KEEP_ALIVE 2
```

```suggestion
ENV WEB_SERVER_KEEP_ALIVE 2
```

I don't think we should make an assumption about what protocol is being used for the keepalive

---

I did HTTP to make it explicit. For uwsgi, it has http_keepalive and so_keepalive for sockets.